systest/backup/advanced-scenarios/acl-nonAcl/backup_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"net/http"
+	"testing"
+
+	e2eCommon "github.com/dgraph-io/dgraph/graphql/e2e/common"
+	utilsCommon "github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+const (
+	restoreLocation = "/data/backups/"
+	backupDst       = "/data/backups/"
+)
+
+func TestAclToNonAclBackupRestore(t *testing.T) {
+	t.Logf("acl(backup cluster) to non-Acl(restore cluster) Test")
+	jwtTokenAlpha1, headerAlpha1 := utilsCommon.GetJwtTokenAndHeader(t, "alpha1", 0)
+	BackupRestore(t, jwtTokenAlpha1, "", headerAlpha1, "alpha1", "alpha2")
+}
+
+func TestNonAclToAclBackupRestore(t *testing.T) {
+	t.Logf("non-acl(backup cluster) to acl(restore cluster) Test")
+	jwtTokenAlpha3, headerAlpha3 := utilsCommon.GetJwtTokenAndHeader(t, "alpha3", 0)
+	BackupRestore(t, "", jwtTokenAlpha3, headerAlpha3, "alpha4", "alpha3")
+}
+
+func BackupRestore(t *testing.T, jwtTokenBackupAlpha string, jwtTokenRestoreAlpha string, header http.Header, backupAlpha string, restoreAlpha string) { //nolint:lll
```

Comment:
break this line instead of the `nolint` comment

Reply:
Resolved.